SimPEG/utils/pgi_utils.py

```diff
@@ -877,10 +894,253 @@ def score_samples_with_sensW(self, X, sensW):
             Log probabilities of each data point in X.
         """
         check_is_fitted(self)
-        X = _check_X(X, None, self.means_.shape[1])
+        X = self._validate_data(X, reset=False)
 
         return logsumexp(self._estimate_weighted_log_prob_with_sensW(X, sensW), axis=1)
 
+    def plot_pdf(
```

Comment:
Got the feeling this should go under the `plot_utils`?

Reply:
hum, it could be. I have not thought about it in that way. the idea was to have it as an object function `gmm.plot_pdf`. Are there specific reasons you would prefer it as `utils.plot_pdf(gmm)`?

Reply:
Oh ok, didn't realize it was under the class. My bad.